es-app/src/MetaData.cpp

```diff
@@ -14,10 +14,10 @@ MetaDataDecl gameDecls[] = {
 	{"marquee",     MD_PATH,                "",                 false,      "marquee",              "enter path to marquee"},
 	{"thumbnail",   MD_PATH,                "",                 false,      "thumbnail",            "enter path to thumbnail"},
 	{"rating",      MD_RATING,              "0.000000",         false,      "rating",               "enter rating"},
-	{"releasedate", MD_DATE,                "not-a-date-time",  false,      "release date",         "enter release date"},
-	{"developer",   MD_STRING,              "unknown",          false,      "developer",            "enter game developer"},
-	{"publisher",   MD_STRING,              "unknown",          false,      "publisher",            "enter game publisher"},
-	{"genre",       MD_STRING,              "unknown",          false,      "genre",                "enter game genre"},
+	{"releasedate", MD_DATE,                "",                 false,      "release date",         "enter release date"},
```

Comment:
TimeUtil.cpp compares with "not-a-date-time" string value - unsure if there are other scenarios where this type of change may break other things. Please assess or at least address this impact if possible.